index.bs

```diff
@@ -2375,12 +2402,64 @@ data-frame = {
   ? 5: media-time ; sync-time
 }
 
+; TODO(jophba): Why do some have type keys, but not others?
 ; type key 120
 video-frame-request = {
  1: uint; encoding-id
- 2: uint; sequence-number: 
+ 2: uint; sequence-number
  ? 3: uint; highest-decoded-frame-sequence-number
 }
 
+streaming-capabilities-request = {
+  request
+}
+
+streaming-capabilities-response = {
+  response
+  1: streaming-capabilities ; streaming-capabilities
+}
+
+streaming-capabilities = {
+  0: [* receive-audio-capability] ; receive-audio
+  1: [* receive-video-capability] ; receive-video
+  2: [* receive-data-capability] ; receive-data
+}
+
+codec = {
+  0: string ; codec-name
+  1: [* codec-parameter] ; parameters
+}
+
+codec-parameter = {
+  0: string ; key
+  1: bytes ; value
+}
+
+receive-audio-capability = {
+  0: codec ; codec
+  ? 1: uint ; max-audio-channels
+  ? 2: float64 ; min-bit-rate
+}
+
+resolution = {
+  0: uint ; horizontal-resolution
+  1: uint ; vertical-resolution
+}
```

Comment:
I'd prefer:

height 
width

That's what it's called everywhere I've seen it

Reply:
Sounds good.